csrc/iter_visitor.cpp

```diff
@@ -987,22 +953,22 @@ std::vector<Val*> InputsOf::outputs(
     Fusion* fusion,
     const std::vector<Val*>& outputs_) {
   InputsOf io;
-  io.traverseTo(fusion, outputs_, false);
+  io.traverseTo(outputs_, false);
```

Comment:
It's a bit strange that the function has a `Fusion` parameter but it isn't used.

Reply:
Done.